src/components/Select/Select.module.css

```diff
@@ -0,0 +1,92 @@
+.select {
+  border-radius: 30px;
+  font-weight: 500;
+  font-size: 28px;
+  padding: 16px 30px 16px 50px;
+  border-radius: 30px;
+  background-color: #000;
+  line-height: 1;
+  position: relative;
+  transition: all 0.1s ease;
+  outline: none;
+  cursor: pointer;
+  user-select: none;
+  &:focus,
+  &:hover {
+    background-color: #ffd400;
+    color: #000;
+    &:before {
+      border-color: #000;
+    }
+  }
+  &:before {
+    content: "";
+    position: absolute;
+    top: 50%;
+    transform: translateY(-50%);
+    left: 20px;
+    width: 12px;
+    transition: all 0.1s ease;
+    height: 12px;
+    border-radius: 50%;
+    border: 4px solid #fff;
```

Comment:
nit:

Let's make `border: 4px solid currentColor` and remove `:hover:before` above?